app/Libraries/Mods.php

```diff
@@ -88,6 +88,17 @@ public function __construct()
         }
     }
 
+    public function checkAllModsMeta(int $rulesetId, array $mods, callable $check, callable $failItemCallback): void
```

Comment:
Everything in this function is external to the class other than the call to a `public $this->mods`, it looks like it should go in `PlaylistItem`